app/views/provider_interface/application_choices/index.html.erb

```diff
@@ -1,6 +1,18 @@
 <% content_for :title, 'Applications' %>
 <% content_for :page_width, 'wide' %>
 
+<% if FeatureFlag.active?('covid_19') %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="app-banner" aria-labelledby="success-message" data-module="govuk-error-summary" role="alert">
+        <div class="app-banner__message">
+          <h2 class="govuk-heading-m" id="success-message">Check our guidance to see how <%= link_to 'coronavirus (COVID-19) will affect your deadlines for responding to applications', '#', class: 'govuk-link' %></h2>
```

Comment:
When are we receiving the link for this? Not comfortable usually merging things with broken links as they do tend to get to live in my experience without being checked...

Reply:
Will wait for https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1714 before merging